### ISSUE TITLE
Uncaught SyntaxError: Identifier 'Price' has already been declared

### ISSUE THREAD

**Issue by emahuni** [hawk-lord/direct-currency-converter-2#25]
This extension is great  Chrome, and I love it. There is an error occuring and annoying me in the console:

```
/Users/em/Library/Application%20Support/Google/Chrome/Default/Extensions/hgdfddnhgpincbchioebogaljemgkpgl/3.2.0_0/content/dcc-functions.js:1 Uncaught SyntaxError: Identifier 'Price' has already been declared
    at /Users/em/Library/Application%20Support/Google/Chrome/Default/Extensions/hgdfddnhgpincbchioebogaljemgkpgl/3.2.0_0/content/dcc-functions.js:1
```
Chrome Version 78.0.3904.108 (Official Build) (64-bit)
MacOS Majove

**Comment by hawk-lord** [comment]
Thanks for letting me know. I'll look into that.

**Comment by hawk-lord** [comment]
By the way, the current version is https://github.com/hawk-lord/direct-currency-converter-3

**Comment by emahuni** [comment]
this is the version I am using right now and it's causing that issue:
![Screenshot 2020-01-02 at 21 40 56](https://user-images.githubusercontent.com/7290508/71688483-996c7900-2da8-11ea-942f-56d24059fe6b.png)

**Comment by emahuni** [comment]
oh sorry I highlighted the version, in case you can't see that it's version 3.2.0

**Comment by hawk-lord** [comment]
I have fixed the issue now in version 3.2.1.
